stronk/schemas/user/mutations.py

```diff
@@ -0,0 +1,79 @@
+import graphene
+from firebase_admin import auth
+from flask import current_app, g
+
+from stronk import db
+from stronk.models.user import User as UserModel
+from stronk.schemas.user.type import User
+from tests import constants as c
+
+class CreateUser(graphene.Mutation):
+    """Create a user."""
+    # declare class attributes
+    user = graphene.Field(User)
+
+    class Arguments:
+        name = graphene.String(required=True)
+        username = graphene.String(required=True)
+        email = graphene.String(required=True)
+        currentProgram = graphene.Int(required=False)
+
+    def mutate(root, info, name, username, email, currentProgram=None):
+        id = c.TEST_ID if current_app.config['ENV'] == 'testing' else auth.get_user(g.get("firebase_token"))
```

Comment:
I think we should be getting uid from the user as well `auth.get_user(g.get("firebase_token")).uid`

Reply:
@13ABEL great that makes things a lot easier

Reply:
@13ABEL  just realized that was a typo and wasn't accessing the uid attribute mbmb